MAINTAINERS.md

```diff
@@ -45,3 +45,9 @@ To become a maintainer, open a pull request to this list.
 * [Fletcher Nichol](https://github.com/fnichol)
 * [Romain Sertelon](https://github.com/rsertelon)
 * [Scott Hain](https://github.com/scotthain)
+
+## 'Veteran' Maintainers
+
+This section has been added for the benefit of those who contribute heavily to the core-plan's and are eligible for 'fastrack' promotion.  See [fastrack policy document](https://github.com/habitat-sh/core-plans/blob/master/docs/dev/policy_documents/merging-and-promoting-conventions.md)
+
+* [Graham Weldon](https://github.com/predominant)
```

Comment:
@predominant: You're the first 'veteran'...

Reply:
@gavindidrichsen May I suggest that you add yourself to the list as well:

```
 > git shortlog -sne --since "3 months ago" | head -10
    66	Graham Weldon <graham@grahamweldon.com>
    32	Gavin Didrichsen <gavin.didrichsen@gmail.com>
    10	W. Duncan Fraser <duncan@wduncanfraser.com>
     7	Tim Smith <tsmith84@gmail.com>
     6	Scott Macfarlane <smacfarlane@chef.io>
     6	Christopher Maher <chris@mahercode.io>
     5	Ben Dang <me@bdang.it>
     4	mwrock <matt@mattwrock.com>
     4	Salim Afiune <afiune@chef.io>
     3	Steven Danna <steve@chef.io>
```

Reply:
Done.